src/components/common/YourCompany/BusinessStartDate.vue

```diff
@@ -155,23 +156,20 @@ export default class StartDate extends Mixins(CommonMixin, DateMixin) {
 
   /** The business date or business start date string. */
   get businessStartDate (): string {
-    // safety check
-    if (this.isApplicableFiling) {
-      if (this.hasBusinessStartDateChanged) {
-        // set the Corrected flag when reloading from a saved filing
-        this.isCorrected = true
-        return this.yyyyMmDdToPacificDate(this.getCorrectionStartDate, true)
-      }
-      if (this.getBusinessStartDate) {
-        return this.yyyyMmDdToPacificDate(this.getBusinessStartDate, true)
-      }
+    if (this.hasBusinessStartDateChanged) {
+      // set the Corrected flag when reloading from a saved filing
+      this.isCorrected = true
+      return this.yyyyMmDdToPacificDate(this.getCorrectionStartDate, true)
+    }
+    if (this.getBusinessStartDate) {
+      return this.yyyyMmDdToPacificDate(this.getBusinessStartDate, true)
     }
     return '(Not entered)'
   }
 
   /** Whether this component is valid. */
   get isValid (): boolean {
-    return (!this.onEditMode && !!this.getCorrectionStartDate)
+    return (!this.onEditMode && (!!this.getCorrectionStartDate || !!this.getBusinessStartDate))
```

Comment:
If there is no correction start date, check business start date (otherwise, initial correction showed invalid business start date).